src/data.ts

```diff
@@ -608,6 +608,13 @@ const protocols: Protocol[] = [
     module: "adapters/nansen.ts",
     portfolioUrl: "https://app.nansen.ai/points",
   },
+  {
+    id: 40,
+    name: "Pjrx",
```

Comment:
typo name, also the full name is 'Project X'
```suggestion
    name: "Project X",
```